### ISSUE TITLE
unlambda: false positive 

### ISSUE THREAD

**Issue by Abirdcfly** [go-critic/go-critic#1362]
```go
func New(options ...string) error {
	return nil
}

var NewWithTimeout = func(options ...string) error {
	return New(append(options, "timeout")...)
}
```

```
unlambda: replace `func(options ...string) error {
        return New(append(options, "timeout")...)
}` with `New` (gocritic)
var NewWithTimeout = func(options ...string) error {
                     ^

```

**Comment by cristaloleg** [comment]
Thanks for the report and sorry for the issue.
